shared/js/background/events.js

```diff
@@ -433,6 +434,8 @@ browserWrapper.createAlarm('clearExpiredHTTPSServiceCache', { periodInMinutes: 6
 browserWrapper.createAlarm('rotateUserAgent', { periodInMinutes: 24 * 60 })
 // Rotate the sessionKey
 browserWrapper.createAlarm('rotateSessionKey', { periodInMinutes: 24 * 60 })
+// Expire site breakage reports
+browserWrapper.createAlarm('clearExpiredBrokenSiteReportTimes', { periodInMinutes: 24 * 60 })
```

Comment:
I think this is not going to fire unless the browser is open for 24 hours. I think a simpler solution will be to just run a clear on extension start, then rerun whenever a broken site report is sent.

Reply:
That doesn't clear the data for local users though which was the main concern. I think on start and every 60 might be a better compromise (I think I left a comment saying it was every 60 anyway and was my intent) I think the one above should do the same too.